core/Core.php

```diff
@@ -136,8 +135,9 @@
 /**
  * Load error handlers
  */
-Debug::loadErrorHandlers();
 
+$errorHandler = Injector::inst()->get('ErrorHandler');
+$errorHandler->start();
```

Comment:
If we are defining a service with a promised method `start` should we define this as an interface somewhere? SilverStripe\Framework\Logging\ErrorHandler?